packages/react/src/converter/react-wrapper.test.tsx

```diff
@@ -79,4 +79,15 @@ describe('CdsTestComponent', () => {
     );
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('correctly passes down className to the underlying component', () => {
+    const wrapper = shallow(
+      <div>
+        <CdsTestComponent className="test-classname">Hello World</CdsTestComponent>
+      </div>
+    );
+
+    const renderedComponent = wrapper.find('ReactWrapperComponent');
+    expect(renderedComponent.prop('className')).toEqual('test-classname');
```

Comment:
I think I would expect the test to look at whether the classname was passed down to the wrapped component. Maybe I'm missing something here? Once the component is rendered, you should be able to look in the DOM and see if the web component has the classname in its classList. I would think that test would fail without the fix.

Reply:
That makes total sense, and that worked indeed! I could verify that the test failed without my change, and passed afterwards. Thanks for the pointer and sorry for being slow, it's the first time I work with web components (and Clarity) so still adjusting.